docs/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,30 @@
+# Proposed Changes
+#Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.:
+
+This ticket was primarily designed to address the review points received when Swift Code for the iOS application "Memorize" was being reviewed.
+The key review point was to follow standard software development practices.
+
+Git was not implemeneted on the previous changes and as there were no version controlling present in the previous code. So, new Repo is created and the review points received in the previously are fixed in this PR. 
+The previous code for which review comments were received is present in another repo https://github.com/KedarDhere/Memorize.
+
+# Jira Stories
+
+https://oregonstate-innovationlab.atlassian.net/jira/software/projects/LAB/boards/1?selectedIssue=LAB-225
+
+# Checklist
+
+[ ] This has been tested
+[ ] Self-documenting code
+[ ] Code is linted
+
+# Instructions to review
+
+Tell me what to do:
+
+Following review comments were received and need to review whether all the received review points are fixed:
+
+	1.Use guard to the functions instead of if wherever possible. 
+	2.Implement Git 
+	3.Use card objects instead of indexes 
+	4.Install SwiftLinter
+
```

Comment:
But this specific to this PR.  This shouldn't be in the template because it doesn't generalize to every PR.